src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public mate.academy.model.CinemaHall add(mate.academy.model.CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public mate.academy.model.CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).orElseThrow(() ->
+                new RuntimeException("cannot find cinema hall with id: " + id));
```

Comment:
```suggestion
                new DataProcessingException("cannot find cinema hall with id: " + id));
```
Create custom DataProcessingException